west.yml

```diff
@@ -92,7 +92,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 39615db4095858d24d87dfd6d8e0970704aae18a
+      revision: pull/40/head
```

Comment:
This needs to be updated before merging.

Reply:
The hal pr has now been merged, so you can update this to the new SHA

Reply:
@MaureenHelm  @henrikbrixandersen i've updated west.yaml with the commit number.

Reply:
Please update the commit message too. `Updated to west.yaml pull 40 on hal_nxp repository.` doesn't make sense anymore

Reply:
changed commit message

Reply:
Sorry to be a pain, but the commit message still isn't right - it's referring to the old hal commit. You could change it to `Updated west.yaml to the hal_nxp commit which enabled entropy drivers for LPC family.`

Reply:
@MaureenHelm done!

Reply:
thank you!